models/articles_models.js

```diff
@@ -46,20 +46,64 @@ exports.patchArticleById = (inc_votes, article_id) => {
     });
 };
 
-exports.fetchArticles = () => {
-  return connection
-    .query(
-      `SELECT articles.*, COUNT(comments.article_id)::INT AS comment_count 
-    FROM articles 
-    LEFT JOIN comments 
-    ON comments.article_id = articles.article_id
-    GROUP BY articles.article_id
-    ORDER BY created_at DESC`
-    )
-    .then((articles) => {
-      return articles.rows;
-    })
-    .catch((err) => {
-      return err;
+exports.fetchArticles = async (
```

Comment:
you've got the `async` keyword here but you're not using `await`!

Either syntax is fine but pick one or the other!